metadata/algolia/jobs.xml

```diff
@@ -114,4 +114,21 @@ Can perform full records updates or partial records updates. See the 'indexingMe
         <triggers/>
     </job>
 
+    <job job-id="AlgoliaContentIndex" priority="0">
+        <description>Index all searchable content assets assigned to the selected site to Algolia.</description>
+        <parameters/>
+        <flow>
+            <context site-id="RefArch"/>
+            <step step-id="algoliaContentIndex" type="custom.algoliaContentIndex" enforce-restart="false">
+                <description>Index all searchable contents assigned to the selected site to Algolia. The list of indexed attributes is configurable.
+It highly depends content search index on salesforce, so do not forget to trigger search index after content update or enable incremental update.</description>
```

Comment:
```suggestion
It highly depends on the content search index on Salesforce, so do not forget to trigger search index after content update or enable incremental update.</description>
```
I tried to update a content asset and re-run the job, and the content was correctly updated in the Algolia record, without doing anything else. Does it means that we have incremental updates on our sandboxes? Where can we enable/disable it?